docs/DeveloperGuide.md

```diff
@@ -236,42 +236,124 @@ _{Explain here how the data archiving feature will be implemented}_
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
-
-*{More to be added}*
+ Priority | As a …​ | I want to... | So that I can... |
+| --- | --- | --- | --- |
+| `* * *` | Tutor with many students | Store my student&#39;s contact info/emails | Contact them easily |
+| `* * *` | Tutor with many classes | Create classes | Put my students in the appropriate classes |
+| `* * *` | Tutor with many classes | Insert my students into the appropriate classes | Organise my students via classes
+| `* * *` | New Tutor | View the help menu | Be familiar with app usage |
+| `* * *` | Tutor | Delete student entries | Update my list of students if a student were to drop the module or change tutorial slots |
+| `* *` | Tutor | View my students&#39; test scores easily | Gauge my teaching efficiency |
+| `* *` | Tutor | Mark my student&#39;s attendance and participation | Gauge each student&#39;s participation level |
+| `* *` | Tutor teaching CS modules | View my students&#39; GitHub | Gauge my student&#39;s progress in their coding assignment |
+| `* *` | Tutor | View all my lessons | Manage my time more efficiently |
+| `* *` | Tutor | Record my feedback for my students | Give participation points |
+| `* *` | Forgetful Tutor | Track all my tasks | Know which is of greater urgency |
+| `* *` | Busy Tutor | Store zoom links | Retrieve these zoom links for my lessons, consultations, etc. |
+| `* *` | Busy Tutor | Store class rooms | Be reminded of my lesson venues |
+| `* *` | Tutor with many students | Keep notes on each student&#39;s performance | Track their progress over time |
+| `* *` | Careless tutor | Undo my commands | Correct any errors when I input things wrongly |
+| `* *` | Tutor | Set recurring events (eg. lessons for every week) | Avoid typing the same events |
+| `* *` | Tutor | Calculate the mean, median, mode of my students' scores | Gauge the overall performance of my class |
+| `* *` | Tutor | Store the questions asked by students | Provide students with answers immediately, for questions that were asked before |
+| `* *` | Tutor | Be reminded of my lessons | Attend them |
+| `* *` | Tutor | Update the information of my students | Update my understanding of the progress of my students |
+| `* *` | Tutor | Categorise my students into how well they are doing | Dedicate more time towards the weaker students |
+| `* *` | Careless Tutor | Redo my undone actions | Easily reverse my accidental undos. |
+| `*` | Caring tutor | Take note of student&#39;s special needs, if any | Cater my teaching toward them |
+| `*` | Tutor for many semesters | Archive my past semesters | Avoid cluttering the app |
+| `*` | Tutor | Store teaching feedback given by my tutees | Improve my teaching |
```

Comment:
Maybe it will be nice to change `tutees` to `students`? Since we are using students at all other places.